### ISSUE TITLE
Rails 7 Active Support issue (Wrong number of arguments (given 3, expected 1..2) (ArgumentError))

### ISSUE THREAD

**Issue by ExReanimator** [pawurb/rails-brotli-cache#7]
Just tried to run my app with the gem and have faced with the following error:

**Wrong number of arguments (given 3, expected 1..2) (ArgumentError)**
rails (0fec9536ca43) [activesupport/lib/active_support/cache/strategy/local_cache.rb in increment](https://github.com/rails/rails/blob/7-0-stable/activesupport/lib/active_support/cache/strategy/local_cache.rb#L93)

-----

my config/environments/staging.rb:

``` ruby
...
config.cache_store = RailsBrotliCache::Store.new(
    ActiveSupport::Cache::RedisCacheStore.new(
      url: ENV['REDIS_URL'] || 'redis://localhost:6379/0/cache',
      connect_timeout: 30,
      read_timeout: 5,
      write_timeout: 5,
      reconnect_attempts: 3,
      error_handler: proc { |method:, returning:, exception:|
                       Sentry.capture_exception exception,
                                                tags: { method: method,
                                                        returning: returning }
                     }
    )
)
...
```
-----

Rails 7.0.4.3

**Comment by pawurb** [comment]
@ExReanimator thanks for spotting. But I cannot reproduce. can you send a full error stack trace?

**Comment by ExReanimator** [comment]
```
rails (0fec9536ca43) activesupport/lib/active_support/cache/strategy/local_cache.rb in increment at line 93
rails-brotli-cache (0.4.1) lib/rails-brotli-cache/store.rb in increment at line 133
rack-attack (6.6.1) lib/rack/attack/cache.rb in do_count at line 74
rack-attack (6.6.1) lib/rack/attack/cache.rb in count at line 27
rack-attack (6.6.1) lib/rack/attack/throttle.rb in matched_by? at line 31
rack-attack (6.6.1) lib/rack/attack/configuration.rb in block in throttled? at line 91
rack-attack (6.6.1) lib/rack/attack/configuration.rb in any? at line 90
rack-attack (6.6.1) lib/rack/attack/configuration.rb in throttled? at line 90
rack-attack (6.6.1) lib/rack/attack.rb in call at line 118
http_accept_language (2.1.1) lib/http_accept_language/middleware.rb in call at line 14
```

**Comment by pawurb** [comment]
@ExReanimator thanks! I've managed to reproduce. It should be fixed in `0.4.2`, please confirm.

**Comment by ExReanimator** [comment]
@pawurb Yep, the problem has been fixed in v0.4.2. Thank you!
